addOns/automation/src/test/resources/org/zaproxy/addon/automation/resources/template-max.yaml

```diff
@@ -64,6 +64,8 @@ jobs:
         method:                        # String: A non-empty request method, default: GET
         data:                          # String: Optional data to send in the request body, supports vars
         responseCode:                  # Int: An optional, expected response code against which the actual response code will be matched
+        headers:                       # An optional list of headers
```

Comment:
Worth adding to `-min` as well.

Reply:
I had a discussion about adding this to `-min` with @psiinon and he mentioned that its okay if we dont add it there.

Reply:
What's the reason not to?

Reply:
The idea was that the min template really was the minimum set of configs needed, and the headers arent needed for many requests

Reply:
maybe because its an optional field and is not required for the minimal working of requestor job?

Reply:
The optional is not the/a reason, there are many optional fields in the `-min`. I'd argue the headers are worth mentioning always but fine.